helm/scripts/multiagent-presync.sh

```
@@ -8,6 +8,9 @@ source "$(dirname "$0")/postsync-lib.sh"
 #create secret
 create_or_update_resource "secret generic" multiagent-be-secret \
   --from-literal=CREDENTIAL_ENCRYPTION_KEY="$CREDENTIAL_ENCRYPTION_KEY" \
+  --from-literal=MCP_AUTH_STATE_SECRET="$MCP_AUTH_STATE_SECRET" \
+  --from-literal=gemini_api_key="$gemini_api_key" \
+  --from-literal=gemini_model_name="$gemini_model_name"
```

Shellcheck (0.11.0)

[warning] gemini_model_name is referenced but not assigned.

(SC2154)